src/services/users.js

```diff
@@ -0,0 +1,8 @@
+import { create } from 'apisauce'
```

Comment:
Other think normally we create a folder config and inside of this create a file api.js where you create or config de apisauce